app/models/course_user.rb

```diff
@@ -3,7 +3,7 @@ class CourseUser < ActiveRecord::Base
   after_initialize :set_defaults, if: :new_record?
   before_validation :set_defaults, if: :new_record?
 
-  enum role: { student: 0, teaching_assistant: 1, manager: 2, owner: 3 }
+  enum role: { student: 0, teaching_assistant: 1, manager: 2, owner: 3, auto_grader: 4 }
```

Comment:
Oh, so these graders are actually evaluators. And when it will be created ? for example, the course_user here, we will have it for each course?